interface/super/edit_layout.php

```diff
@@ -1600,7 +1632,13 @@ function myChangeCheck() {
 	{id: 'U',text:'" . xla('Capitalize all') . "'},
 	{id: 'V',text:'" . xla('Vendor') . "'},
 	{id: 'X',text:'" . xla('Do Not Print') . "'},
-	{id: '0',text:'" . xla('Read Only') . "'},
+    {id:'grp',text:'" . xla('Stylings') . "',children:[
+        {id: 'RS',text:'" . xla('Add Bottom Border Row') . "'},
+        {id: 'RO',text:'" . xla('Outline Entire Row') . "'},
+        {id: 'DS',text:'" . xla('Add Data Bottom Border') . "'},
+        {id: 'DO',text:'" . xla('Outline Data Col') . "'}
+    ]},
```

Comment:
Nice options. As I recall, somebody was asking how to do this recently on forums or chat.

Reply:
To clarify, are 2 digit options supported. For example, will this also turn on the 'D' and 'o' and 'R' and 'S' options when used downstream.

Reply:
It's the reason I set these up as json. But nowww you have me wary, I better check again:)

Reply:
Sure enough, guess I have to get creative...